generators/gulp/index.js

```diff
@@ -47,14 +47,15 @@ module.exports = generators.Base.extend({
 
       extend(pkg, {
         devDependencies: {
-          gulp: '^3.9.0',
+          gulp: '3.9.1',
+          'gulp-coveralls': '^0.1.4',
```

Comment:
I think that's the wrong place to bump it. We only want to add gulp-coveralls if we're using coveralls.